### PR TITLE
Documentation: Spanish translation of fast_tokenizers.mdx

### PR DESCRIPTION
**Pull request by jloayza10**
# What does this PR do?

Fixes #15947
Translation to Spanish of fast_tokenizers.mdx

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? #15947

### CODE REVIEW

**Comment by omarespejel** [comment]
Thank you @jloayza10! Could you please add `fast_tokenizers` to [`transformers/docs/source/es/_toctree.yml`](https://github.com/huggingface/transformers/blob/main/docs/source/es/_toctree.yml)? As a reference, you can use the [new Translation](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md) guide (section "✍️ Start translating"). This would allow the tests to pass.

**Comment by jloayza10** [comment]
Hi @omarespejel , I added `fast_tokenizers` to the _toctree.yml file.
There is still a `check_code_quality` check that is failing, are there further modifications on my part to perform to address this or is this ok?

File: docs/source/es/fast_tokenizers.mdx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Usa los tokenizadores de 🤗 Tokenizers
+
+[`PreTrainedTokenizerFast`] depende de la librería [🤗 Tokenizers](https://huggingface.co/docs/tokenizers). Los tokenizadores obtenidos desde la librería 🤗 Tokenizers pueden ser 
+cargados muy simplemente a 🤗 Transformers.
```
Comment (omarespejel) [inline docs/source/es/fast_tokenizers.mdx:16]:
```suggestion
cargados de forma muy sencilla en los 🤗 Transformers.
```

File: docs/source/es/fast_tokenizers.mdx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+Antes de entrar en los detalles, podemos crear un tokenizador dummy en unas cuantas líneas:
```
Comment (omarespejel) [inline docs/source/es/fast_tokenizers.mdx:18]:
```suggestion
Antes de entrar en detalles, comencemos creando un tokenizador dummy en unas cuantas líneas:
```

File: docs/source/es/fast_tokenizers.mdx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+>>> tokenizer = Tokenizer(BPE(unk_token="[UNK]"))
+>>> trainer = BpeTrainer(special_tokens=["[UNK]", "[CLS]", "[SEP]", "[PAD]", "[MASK]"])
+
+>>> tokenizer.pre_tokenizer = Whitespace()
+>>> files = [...]
+>>> tokenizer.train(files, trainer)
+```
+
+Ahora tenemos un tokenizador entrenado con los archivos (files en el código) que fueron definidos. Lo podemos seguir utilizando en ese entorno de ejecución (runtime en inglés), o se lo puede guardar
```
Comment (omarespejel) [inline docs/source/es/fast_tokenizers.mdx:34]:
```suggestion
Ahora tenemos un tokenizador entrenado en los archivos que definimos. Lo podemos seguir utilizando en ese entorno de ejecución (runtime en inglés), o puedes guardarlo
```

File: docs/source/es/fast_tokenizers.mdx [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+en un archivo JSON para reutilizarlo en un futuro.
+
+## Cargando directamente desde el objeto tokenizador 
+
+Veamos como utilizar este objeto tokenizador en la librería 🤗 Transformers. La clase
```
Comment (omarespejel) [inline docs/source/es/fast_tokenizers.mdx:39]:
```suggestion
Veamos cómo utilizar este objeto tokenizador en la biblioteca 🤗 Transformers. La clase
```

File: docs/source/es/fast_tokenizers.mdx [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+[`PreTrainedTokenizerFast`] permite una instanciación simple, al aceptar el objeto
```
Comment (omarespejel) [inline docs/source/es/fast_tokenizers.mdx:40]:
```suggestion
[`PreTrainedTokenizerFast`] permite una instanciación fácil, al aceptar el objeto
```

**Review by omarespejel — CHANGES_REQUESTED** [review]

**Review by omarespejel — APPROVED** [review]

**Comment by omarespejel** [comment]
Thank you very much @jloayza10! This was great. Please let me know if you would like to translate another file 🤗

**Pull request merged**